server/liveblog/items/items.py

```diff
@@ -77,6 +78,16 @@ def on_create(self, docs):
         for doc in docs:
             update_dates_for(doc)
             doc['original_creator'] = str(get_user().get('_id'))
+            if doc.get('item_type'):
```

Comment:
can we have just one if here ? the imbrication is not so nice
`if doc.get('item_type','') == 'embed': `